module_evaluations/TCR-6_2022-09-28.MD

```diff
@@ -0,0 +1,61 @@
+# Module acceptance criteria template
+
+## How to use this form
+When performing a technical evaluation of a module, create a copy of this document and use the conventions below to indicate the status of each criteria.  The evaluation results should be placed in the [module_evaluations](https://github.com/folio-org/tech-council/tree/master/module_evaluations) directory and should conform to the following naming convention:  `{JIRA Key}_YYYY-MM-DD.MD`, e.g. `TCR-1_2021-11-17.MD`.  The date here is used to differenciate between initial and potential re-evaluation(s).  It should be the date which the evaluation results file was created.
+
+* [x] ACCEPTABLE
+* [x] ~INAPPLICABLE~
+* [ ] UNACCEPTABLE
+  * comments on what was evaluated/not evaluated, why a criterion failed
+
+## [Criteria](https://github.com/folio-org/tech-council/blob/4dc5ac6d632fb880131cd214d212f973683b9ebe/MODULE_ACCEPTANCE_CRITERIA.MD)
+
+* [X] Upon acceptance, code author(s) agree to have source code canonically in folio-org github
+* [X] Copyright assigned to OLF
+* [X] Uses Apache 2.0 license
+* [X] Third party dependencies use an Apache 2.0 compatible license
+* [X] Module’s repository includes a compliant Module Descriptor
+* [X] Modules must declare all consumed interfaces in the Module Descriptor “requires” and “optional” sections
+* [ ] Environment vars are documented in the ModuleDescriptor - _note: read more at [https://wiki.folio.org/pages/viewpage.action?pageId=65110683](https://wiki.folio.org/pages/viewpage.action?pageId=65110683)_
+  * The following documented environment variables are not present in the module descriptor: `EVENTS_PUBLISHER_BOOTSTRAP_SERVERS`, `EVENTS_CONSUMER_BOOTSTRAP_SERVERS`, `EVENTS_PUBLISHER_ZK_CONNECT`, `EVENTS_CONSUMER_ZK_CONNECT`
+* [X] Back-end modules must define endpoints consumable by other modules in the Module Descriptor “provides” section
+* [ ] All API endpoints are documented in RAML or OpenAPI
+  * This work is being done by the developers, but it is not yet complete.
+* [X] All API endpoints protected with appropriate permissions
+* [X] No excessive permissions granted to the module
+* [X] Code of Conduct statement in repository
+* [ ] Installation documentation included
+  * Documentation which assits an implementor in getting this module runnin locally would satisfy this requirement
```

Comment:
What was the reason for failing this criteria?

Reply:
What additional documentation would you expect specifically for a local deployment?

Maybe this is another criteria we want to evaluate in the working group.